askama_shared/src/lib.rs

```diff
@@ -350,4 +384,23 @@ mod tests {
 
         let _config = Config::new(raw_config);
     }
+
+    #[test]
+    fn escape_modes() {
```

Comment:
I see this example of adding a new escape mode, is there an example of showing how to define those new escapes?

For example, defining a new custom escape for JS, and then testing that it is properly escaped.